src/main/java/org/example/Logging.java

```diff
@@ -0,0 +1,10 @@
+package org.example;
+
+import org.apache.log4j.Logger;
+
+public class Logging {
+    private static final Logger log = Logger.getLogger(Logging.class);
+    public static void loggerAdding(String forLog){
```

Comment:
forLog - как это перевести?
название метода не по конвенции

```suggestion
    public static void addLog(String logText){
```